### PR TITLE
Add shinytest2

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: inst/unitTests/test_shinyApp.R [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+test_shinyApp <- function()
+{
+  message(sprintf("--- test_shinyApp"))
+  
+  options(chromote.timeout = 60)
+  shinytest2::load_app_env()
+  
+  #test_that("{shinytest2} recording: test_app", {
+    
+    app <- AppDriver$new(app_dir = system.file("app", package = "igvShiny"), name = "test_app", height = 695, width = 1235)
```
Comment (gladkia) [inline inst/unitTests/test_shinyApp.R:19]: please use the currently available demo with sth like:
```
sf <- system.file(package = "igvShiny", "demos", "igvShinyDemo.R")
app <- shinytest2::AppDriver$new(shiny::shinyAppFile(sf)
```

File: inst/app/app.R [hunk 2]
```diff
@@ -0,0 +1,237 @@
+library(igvShiny)
```
Comment (gladkia) [inline inst/app/app.R:1]: please use the currently available demo (see previous comment).

File: inst/unitTests/test_shinyApp.R (hunk 1 above)
Comment (karollayna) [reply inst/unitTests/test_shinyApp.R]: done

File: inst/app/app.R (hunk 2 above)
Comment (karollayna) [reply inst/app/app.R]: ok

**Pull request merged**